duffel_api/models/offer.py

```diff
@@ -221,6 +221,26 @@ def from_json(cls, json: dict):
         )
 
 
+@dataclass
+class OfferSliceSegmentStop:
+    id: str
```

Comment:
📝 Would you mind add doc string here like the other classes?

Reply:
Done